scaffold_test.go

```diff
@@ -42,7 +42,7 @@ var _ = BeforeSuite(func() {
 		fmt.Println("BEARER_JWT_TOKEN=eyJhbGciOiJSUzI1NiJ9.eyJqdGkiOiIwMDgwNWNlYi0yNzI5LTQ2OTgtYWNiMy1jNTRkZmIzMWM4MjEiLCJzdWIiO\n")
 		fmt.Println("NOTE:-")
 		fmt.Println("BEARER_JWT_TOKEN can be gotten by `get_token -u sramamoorthy+edgetest@apigee.com:Test12345`")
```

Comment:
This line (line 44) is the one that I think needs changing. And if that is a real password then you should also change that password now because it's now public.